tests/functional/modules/test_zos_find_func.py

```diff
@@ -298,6 +303,8 @@ def test_find_vsam_in_volume(ansible_zos_module):
             batch=[dict(name=i, state='absent') for i in VSAM_NAMES]
         )
         hosts.all.zos_data_set(name=alternate_vsam, state='absent')
+        volumes.free_vol(volume_1)
```

Comment:
Should these be in a finally block in case the test fails and they don't get freed?